presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -681,3 +681,44 @@ The following properties allow tuning the :doc:`/functions/regexp`.
     to hit the limit on matches for subsequent rows as well, you want to use the
     correct algorithm from the beginning so as not to waste time and resources.
     The more rows you are processing, the larger this value should be.
+
+
+Logging Properties
+------------------
+
+Presto uses the server application framework `Airlift
+<https://github.com/airlift/airlift>`_ and therefore supports its configuration
+properties for the HTTP server and general logging.
+
+``http-server.log.max-history``
```

Comment:
Also document `http-server.log.enabled`, `http-server.log.path`

(`HttpServerConfig` in airlift)

Reply:
added